src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface MovieCardProps {
+  movie: Movie
+}
```

Comment:
in videos Misha said its better to use type instead of interface for components props, its more flexible

Reply:
honestly, it's a subjective thing, you may use what you like